Dockerfile

```
@@ -1,20 +1,21 @@
 FROM node:18.18.0-alpine AS base
-
 WORKDIR /app
 
+RUN npm i -g pnpm
```

Have you experienced how sometimes the single-letter parameters stop being obvious after a few months?
```suggestion
RUN npm install --global pnpm
```